New quarkus-maven-plugin related warnings for CI builds using quarkus main branch

We see some Mandrel IT tests fail due to new warnings appearing in the quarkus main branch relating to the quarkus-maven-plugin:

Symptom looks like:
```
2025-11-21T01:56:45.7095600Z [ERROR] Tests run: 3, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 625.435 s <<< FAILURE! - in org.graalvm.tests.integration.JFRTest
2025-11-21T01:56:45.7096752Z [ERROR] jfrPerfTest{TestInfo}  Time elapsed: 435.076 s  <<< FAILURE!
2025-11-21T01:56:46.0386739Z [ERROR] Failures: 
2025-11-21T01:56:46.0389618Z [ERROR]   JFRTest.jfrPerfTest:186->jfrPerfTestRun:235 build-and-run.log log should not contain error or warning lines that are not whitelisted. See /home/runner/work/mandrel/mandrel/mandrel-integration-tests/testsuite/target/archived-logs/org.graalvm.tests.integration.JFRTest/jfrPerfTest/build-and-run.log and check these offending 1 lines: 
2025-11-21T01:56:46.0395735Z [ERROR] Tests run: 22, Failures: 1, Errors: 0, Skipped: 2
2025-11-21T01:56:46.0435133Z [INFO] testsuite .......................................... FAILURE [37:44 min]
2025-11-21T01:56:46.0452785Z [ERROR]   mvn <args> -rf :testsuite
2025-11-21T01:56:46.0591517Z ##[error]Process completed with exit code 1.
```

Actually caused by new warnings:
```
 2025-11-21 01:56:45.676 INFO  [o.g.t.i.u.Commands] (pidKiller) Killing PID: 14430, forcefully: true
Error:  Tests run: 3, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 625.435 s <<< FAILURE! - in org.graalvm.tests.integration.JFRTest
Error:  jfrPerfTest{TestInfo}  Time elapsed: 435.076 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: 
build-and-run.log log should not contain error or warning lines that are not whitelisted. See /home/runner/work/mandrel/mandrel/mandrel-integration-tests/testsuite/target/archived-logs/org.graalvm.tests.integration.JFRTest/jfrPerfTest/build-and-run.log and check these offending 1 lines: 
Warning:  The Maven extensions for the Quarkus Maven plugin are not enabled for this build. We recommend enabling this, so that the plugin can verify essential build settings have been configured as required. Please enable by adding "<extensions>true</extensions>" in your quarkus-maven-plugin declaration; it should look like: ==> expected: <true> but was: <false>
	at org.graalvm.tests.integration.JFRTest.jfrPerfTestRun(JFRTest.java:235)
	at org.graalvm.tests.integration.JFRTest.jfrPerfTest(JFRTest.java:186)
```

We need to account for that.

--- a/apps/jfr-native-image-performance/pom.xml
+++ b/apps/jfr-native-image-performance/pom.xml
@@ -63,6 +63,7 @@
                 <groupId>${quarkus.platform.group-id}</groupId>
                 <artifactId>quarkus-maven-plugin</artifactId>
                 <version>${quarkus.version}</version>
+                <extensions>true</extensions>
                 <executions>
                     <execution>
                         <goals>

